templates/bot/csharp/notification-function-base/GettingStarted.txt

```diff
@@ -0,0 +1,53 @@
+Welcome to the Teams Toolkit! Here are some tips to get started building your app.
+
+Prerequisites
+----------
+
+To debug a Bot capability in a Teams app, you'll need to setup Ngrok(https://ngrok.com/). Ngrok is used to help forward external messages from Azure Bot Framework to your locally running machine.
+
+After installing ngrok, use a Command Prompt to run this command: ngrok http 5130
+
+
+Quick start
+----------
+
+Before you run or Start Debugging for the first time, follow these steps:
+
+1. This is a Bot project, so first use a Command Prompt to run this command: ngrok http 5130.
+
+2. Select the Project > Teams Toolkit > Configure Microsoft Teams app menu in Visual Studio. When prompted, sign in using your M365 account.
```

Comment:
Update the command name to keep consistent with the latest UI.

Reply:
Waiting for the existing template change... [pending]